source/locales.tex

```diff
@@ -444,7 +444,7 @@
 components can be instantiated.
 A template parameter with name
 \tcode{International}
-represents the set of all possible specializations on a bool parameter.
+represents the set of all possible specializations on a \tcode{bool} parameter.
```

Comment:
This sentence (and the ones before) appear to be wrong and confusing. I think what they mean are that the name `C` when appearing as a template *argument* (not parameter) in [tab:locale.spec] means each type in a particular set, and the name `International` means both `true` and `false`. But this is an awful way to express that.

Let's fix this more properly:

1) Use `\exposid` for these placeholder identifiers `C` and `International`
2) Remove paragraph 6 and add to paragraph 4:

> An implementation is required to provide those specializations for facet templates identified as members of a category, and for <del>those</del> <ins>those specializations of the forms</ins> shown in [tab:locale.spec]<ins>, where:</ins>
>   * <ins>*`InputIterator`* denotes any type that meets the Cpp17InputIterator requirements,</ins>
>   * <ins>*`OutputIterator`* denotes any type that meets the Cpp17OutputIterator requirements,</ins>
>   * <ins>*`C`* denotes any type from an implementation-defined set of character types, including at least `char` and `wchar_t`, that meet the requirements for a character on which any of the iostream components can be instantiated, and</ins>
>   * <ins>*`International`* denotes any value of type `bool`</ins>.